tactics/declare.ml

```diff
@@ -342,14 +341,14 @@ let declare_variable ~name ~kind d =
         secdef_type = de.proof_entry_type;
       } in
       let () = Global.push_named_def (name, se) in
-      Decl_kinds.Explicit, de.proof_entry_opaque,
+      Glob_term.Explicit, de.proof_entry_opaque,
       poly
   in
   Nametab.push (Nametab.Until 1) (Libnames.make_path DirPath.empty name) (GlobRef.VarRef name);
-  add_section_variable ~name ~kind:impl ~poly;
+  add_section_variable ~name ~poly;
   Decls.(add_variable_data name {opaque;kind});
   add_anonymous_leaf (inVariable ());
-  Impargs.declare_var_implicits name;
+  Impargs.declare_var_implicits ~kind:impl name;
```

Comment:
BTW it would make sense to name the argument `impl` or some similar thing instead of `kind`, avoiding confusion with the `kind` used 2 lines above for `add_variable_data`

Reply:
I agree, done.